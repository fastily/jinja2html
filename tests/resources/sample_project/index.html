{% extends 'templates/base.html' %}

{% set title = "index" %}

{% block content %}
{% raw %}
<div class="container">
    <h1>Hello, World! This is the index page.</h1>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Enim nihil tenetur obcaecati iste sunt voluptatem quidem, repellat fugiat quam minima, deserunt corporis esse atque natus? Qui eligendi blanditiis facilis amet.</p>

    <ul>
        <li><a href="content1.html">content 1</a></li>
        <li><a href="content2.html">content 2</a></li>
        <li><a href="sub/index.html">example subpage</a></li>
    </ul>
</div>
{% endraw %}
{% endblock %}