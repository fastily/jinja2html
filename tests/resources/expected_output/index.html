<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    

    <title>jinja2html test - index</title>
</head>

<body>
    <header>
        <nav class="navbar navbar-expand-lg navbar-light bg-light">
            <div class="container-fluid">
                <a class="navbar-brand" href="index.html">jinja2html</a>
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
                    <div class="navbar-nav">
                        <a class="nav-link active" aria-current="page" href="#">Home</a>
                        <a class="nav-link" href="#">Features</a>
                        <a class="nav-link" href="#">Pricing</a>
                        <a class="nav-link disabled">Disabled</a>
                    </div>
                </div>
            </div>
        </nav>
    </header>

    

<div class="container">
    <h1>Hello, World! This is the index page.</h1>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Enim nihil tenetur obcaecati iste sunt voluptatem quidem, repellat fugiat quam minima, deserunt corporis esse atque natus? Qui eligendi blanditiis facilis amet.</p>

    <ul>
        <li><a href="content1.html">content 1</a></li>
        <li><a href="content2.html">content 2</a></li>
    </ul>
</div>



    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
    
</body>

</html>